IntelliJ Configuration.txt

```diff
@@ -0,0 +1,31 @@
+Pre=requisites:
+Download tomcat 8 and drop files in to C:\Dev\tomcat (atleast put tomcat folder in same directory as repo).
```

Comment:
Please either generalize this for other operating systems, or add other documentation for Linux and MacOS

Reply:
This is the dev branch. This txt file is just for informational purposes if someone on engine wants to use intellij. Regardless I believe the steps are basically the same for other operating systems. This isn't an official documentation and we aren't prepared to provide one yet.